### PR TITLE
implemented tickets 3.1 & 3.2 as requested

### PR DESCRIPTION
**Pull request by osman-jft**
implemented tickets 3.1 & 3.2 as requested

3.1.create two tables teacher and subjects(one to many)
3.2.create a post api to store teachers and their subjects. Sample request
{ "name" : "teacher", "subjects":["English","Maths"] }

### CODE REVIEW

File: src/main/java/com/example/sb_assign_16_05_23/controller/TeacherController.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@RestController
+@RequestMapping("/api")
+public class TeacherController {
+
+    //teacher controller to create endpoint /api/teachers
+
+    @Autowired
+    TeacherService teacherService;
+
+    @GetMapping("/teachers")
```
Comment (jaineetsingh-jft) [inline src/main/java/com/example/sb_assign_16_05_23/controller/TeacherController.java]: add common url on class level

File: src/main/java/com/example/sb_assign_16_05_23/config/TeacherConfig.java
Comment (jaineetsingh-jft) [inline src/main/java/com/example/sb_assign_16_05_23/config/TeacherConfig.java:1]: No need for separate model mapper class for each dto

**Review by jaineetsingh-jft — CHANGES_REQUESTED** [review]
Please review

File: src/main/java/com/example/sb_assign_16_05_23/controller/TeacherController.java (hunk 1 above)
Comment (osman-jft) [reply src/main/java/com/example/sb_assign_16_05_23/controller/TeacherController.java]: i have added change.

File: src/main/java/com/example/sb_assign_16_05_23/config/TeacherConfig.java
Comment (osman-jft) [reply src/main/java/com/example/sb_assign_16_05_23/config/TeacherConfig.java:1]: added change

**Review by jaineetsingh-jft — APPROVED** [review]

**Pull request merged**